R/utils-add_p_tests.R

```diff
@@ -27,7 +27,7 @@ add_p_test_kruskal.test <- function(data, variable, by, ...) {
 
```

Comment:
This is OK but as.numeric() should be added to the paired version of wilcox.test(), in the definition of add_p_tbl_summary_paired.wilcox.test(). I have tried to do it from rstudio but I don't know how to push to this branch (someday I'll learn how to use git)